### DISCUSSION TITLE
Is it normal that movies libraries complete way faster than tv libraries?

### DISCUSSION THREAD

**Discussion by djnield** [Taxel/PlexTraktSync#611] [Q&A] [answered]
Just asking if it's expected behaviour that TV libraries complete much slower over movie libraries. These are the results once everything is cached. See my TV Show library is just vastly different to anything else.

I notice these indicators when each library is running; 

`Movies [150 it/s]` (_items per second?_)
`TV Shows [0 it/s]`

UHD Movies processed in 11.7 seconds (1300 files)
Movies processed in 2 min 52.7 seconds (28,000 files)
UHD TV processed in 11 min 16.0 seconds (497 shows)
Anime processed in **41 min** 8.1 seconds (900 shows)
Kids TV processed in **67 min** 31.6 seconds (900 shows)
TV Shows processed in **542 min** 8.2 seconds (10,000 shows)

**Comment by glensc** [comment, marked as answer]
duplicate of https://github.com/Taxel/PlexTraktSync/discussions/469

**Comment by glensc** [comment]
Shows progressbar now shows rate of episodes not shows
- https://github.com/Taxel/PlexTraktSync/pull/1578
